doc/display.xml

```diff
@@ -120,7 +127,7 @@ gap> FileString("dot/k4.dot", DotDigraph(gr));
 
 <#GAPDoc Label="DotSymmetricDigraph">
   <ManSection>
-    <Attr Name="DotSymmetricDigraph" Arg="digraph"/>
+    <Oper Name="DotSymmetricDigraph" Arg="digraph"/>
```

Comment:
If `DotDigraph` is an `Attr`, shouldn't `DotSymmetricDigraph` also be an attribute (again, not mutable)?